lib/Rex/Commands/Run.pm

```diff
@@ -255,7 +255,7 @@ sub run {
 
     if ( $args && ref($args) eq "ARRAY" ) {
       my $quoter = Net::OpenSSH::ShellQuoter->quoter( $exec->shell->name );
-      $cmd = "$cmd " . join( " ", map { $_ = $quoter->quote($_) } @{$args} );
+      $cmd = "$cmd " . join( " ", map { $_ = $quoter->quote($_) } @{$args} ); ## no critic ProhibitMutatingListFunctions
```

Comment:
`@args` isn't used in the remainder of the function, which leads to a similar argument as above:

The code as it is there now, is equivalent to:

```perl
@args = map { "$suffix:$_" } @args;
```

due to the fact that the mutated value of `@args` isn't used. I think it's better, even for the scope of this PR not to suppress but to fix this instance.